src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
```

Comment:
The selector `&--#{$i} &__star:nth-child(-n + #{$i})` might be incorrect if the intention is to target a specific number of stars within a `.stars` block. Ensure that the BEM naming convention is correctly applied and that the selector targets the intended elements.